core/src/main/resources/plugins_min_versions.txt

```diff
@@ -12,6 +12,7 @@ plsql=3.4.1.2576
 python=1.14.0.3086
 rpg=2.3.0.1187
 ruby=1.5.0.315
+secrets=1.0
```

Comment:
With the latest changes I made in SLCORE, this addition is no more necessary. You can safely remove this line.